src/support/generation.c

```diff
@@ -357,10 +357,10 @@ __wt_session_gen_enter(WT_SESSION_IMPL *session, int which)
     WT_ASSERT(session, session->id < S2C(session)->session_array.cnt);
 
     /*
-     * Assign the thread's resource generation and publish it, ensuring threads waiting on a
-     * resource to drain see the new value. Check we haven't raced with a generation update after
-     * publishing, we rely on the published value not being missed when scanning for the oldest
-     * generation and for draining.
+     * Assign the thread's resource generation, ensuring threads waiting on a resource to drain see
+     * the new value. Check we haven't raced with a generation update after assigning, we rely on
+     * the published value not being missed when scanning for the oldest generation and for
```

Comment:
"published" is the wrong word

Reply:
This goes back to "publish" meaning a variable shared in memory, can you suggest a better wording?

Reply:
What about just "the new value"

Reply:
aca63526d